.gitignore

```diff
@@ -3,3 +3,6 @@ composer.lock
 phpstan.neon
 phpunit.xml
 vendor
+
+composer.phar
+.idea/
```

Comment:
Please revert this change.

Reply:
Reverted in d18e1d10fcd4f2768afcef81c011bcc29ce0c1e1